backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5701100_AD_WF_Approval_Request.sql

```diff
@@ -0,0 +1,1417 @@
+-- Table: AD_WF_Approval_Request
```

Comment:
why do we need this table here?
asking because `AD_WF_Approval_Request` comes with a lot of java code changes regarding document workflow,
and without them, this table makes no sense.

So the question is: do we need `AD_WF_Approval_Request` feature here?